agent/structs/structs.go

```diff
@@ -1867,6 +1870,9 @@ type IndexedServiceTopology struct {
 type ServiceTopology struct {
 	Upstreams   CheckServiceNodes
 	Downstreams CheckServiceNodes
+
```

Comment:
Should any of these be `json: ",omitempty"`?

Reply:
I don't really see the need since they're not exposed directly via the HTTP API.